README.md

```diff
@@ -67,13 +67,17 @@ If you don't want to display comments for a particular post you can disable them
 
 ### Enabling Google Analytics
 
-To enable Google Anaytics, add the following lines to your Jekyll site:
+To enable Google Anaytics tracking, add or uncomment the following lines in your `_config.yml` and copy-paste your own tracking id:
```

Comment:
@ashmaroli I added example values in the provided `_config.yml`, they are commented by default. We could add them also in the default `_config.yml` shipped with Jekyll.